lifecycle/lifecycle/auth/authorize.py

```diff
@@ -147,51 +147,51 @@ def has_scope_permission(
 
 
 @db_access
-def list_permitted_fatmen(
+def list_permitted_jobs(
     auth_subject: models.AuthSubject,
     scope: str,
-    all_fatmen: List[FatmanDto],
-) -> List[FatmanDto]:
+    all_jobs: List[JobDto],
+) -> List[JobDto]:
     """
-    List fatmen that auth subject has permissions to access.
-    Expand permissions for all resources and whole fatman families,
-    map them to list of individual fatmen from a database.
+    List jobs that auth subject has permissions to access.
+    Expand permissions for all resources and whole job families,
+    map them to list of individual jobs from a database.
     :param scope: Scope of the access (see AuthScope)
-    :return: List of fatmen that auth subject has permissions to access (with no duplicates)
+    :return: List of jobs that auth subject has permissions to access (with no duplicates)
     """
     subject_filter = Q(auth_subject=auth_subject)
     scope_filter = Q(scope=scope) | Q(scope=AuthScope.FULL_ACCESS.value)
     queryset = models.AuthResourcePermission.objects.filter(
         subject_filter & scope_filter
     )
 
-    id_to_fatman = {f'{f.name} v{f.version}': f for f in all_fatmen}
+    id_to_job = {f'{f.name} v{f.version}': f for f in all_jobs}
     family_to_ids = defaultdict(list)
-    for fatman in all_fatmen:
-        family_to_ids[fatman.name].append(f'{fatman.name} v{fatman.version}')
+    for job in all_jobs:
+        family_to_ids[fatman_name].append(f'{job_name} v{job_version}')
```

Comment:
Somethig's messed up. I think it should be
```suggestion
        family_to_ids[job.name].append(f'{job.name} v{job.version}')
```